Casks/goneovim.rb

```diff
@@ -0,0 +1,29 @@
+cask "goneovim" do
+  version "0.4.11"
+
+  if MacOS.version <= :big_sur
+    sha256 "8d9d08db2f96e32944f41c9e7de29de83581deab607b199b3a1516cffcc234e4"
+    url "https://github.com/akiyosi/goneovim/releases/download/v#{version}/Goneovim-#{version}-macos11.tar.bz2"
+    app "Goneovim-#{version}-macos11/goneovim.app"
+  else
+    sha256 "b33f1ab2038424263fd0d49058b16d43ce3eabd56a603c20b24aaed24295f051"
+    url "https://github.com/akiyosi/goneovim/releases/download/v#{version}/Goneovim-#{version}-macos.tar.bz2"
+    app "Goneovim-#{version}-macos/goneovim.app"
+  end
```

Comment:
```suggestion
  if MacOS.version < :big_sur
    sha256 "b33f1ab2038424263fd0d49058b16d43ce3eabd56a603c20b24aaed24295f051"
    url "https://github.com/akiyosi/goneovim/releases/download/v#{version}/Goneovim-#{version}-macos.tar.bz2"
    app "Goneovim-#{version}-macos/goneovim.app"
  else
    sha256 "8d9d08db2f96e32944f41c9e7de29de83581deab607b199b3a1516cffcc234e4"
    url "https://github.com/akiyosi/goneovim/releases/download/v#{version}/Goneovim-#{version}-macos11.tar.bz2"
    app "Goneovim-#{version}-macos11/goneovim.app"
  end
```
`MacOS.version <= :big_sur` is always true (unless you're running the beta macOS 12, in which case you get the one built for versions earlier than Big Sur, which isn't what you want).

Reply:
Probably should match other casks and use `<= :catalina`

Reply:
Suggestion revised.